subsys/bluetooth/mesh/CMakeLists.txt

```diff
@@ -39,9 +39,12 @@ zephyr_library_sources_ifdef(CONFIG_BT_MESH_PROVISIONER provisioner.c)
 
 zephyr_library_sources_ifdef(CONFIG_BT_MESH_PB_ADV pb_adv.c)
 
-zephyr_library_sources_ifdef(CONFIG_BT_MESH_PB_GATT pb_gatt.c)
+zephyr_library_sources_ifdef(CONFIG_BT_MESH_PB_GATT
+    pb_gatt.c
+    pb_gatt_srv.c
```

Comment:
@trond-snekvik Maybe we add `CONFIG_BT_MESH_PB_GATT_SRV` to replace `CONFIG_BT_MESH_PB_GATT`, and define another `CONFIG_BT_MESH_PB_GATT_CLI`, but it's best to solve it after the proxy client is implemented.

Reply:
It can probably wait until it makes sense. It's also an API change of sorts, so it might not be worth doing